.github/workflows/sync-sendgrid-template.yml

```diff
@@ -0,0 +1,14 @@
+on:
+  workflow_dispatch:
+    inputs:
+      forceSyncAll:
```

Comment:
Is there a good reason for having this input use camelCase while the other inputs (in different files) use `FORCE_SYNC_ALL`. Is it purely to separate between `FORCE_SYNC_ALL` and `forceSyncAll`?

Reply:
No need to separate them. I've updated them to be consistent